helper/definitions.py

```diff
@@ -0,0 +1,25 @@
+from enum import Enum
+
+
+class Phases(str, Enum):
+    PREPARATION = 'preparation'
+    EXECUTION = 'execution'
+    LAPSE = 'lapse'
+
+
+class PreparationDialogs(str, Enum):

```

Comment:
```suggestion
class InterventionComponents(str, Enum):

```
Maybe this makes sense in relation to the database naming?